docs/rst/tutorials/micro/user_api/user_api_parameter_server.rst

```diff
@@ -0,0 +1,341 @@
+.. _tutorials_micro_user_api_parameter_server:
+
+Parameter Server
+=======================
+
+.. contents:: Table of Contents
+    :depth: 2
+    :local:
+    :backlinks: none
+
+ROS 2 parameters allow the user to create variables on a node and manipulate/read them with different ROS 2 commands. Further information about ROS 2 parameters can be found on :ref:`Understanding parameters <ROS2Params>`.
+
+micro-ROS parameters implementation has been upgraded in the latest micro-ROS Humble distribution this tutorial will focus on micro-ROS Humble and beyond:
+
+General implementation
+----------------------
+
+Initialization
+^^^^^^^^^^^^^^
+- Default initialization:
+
+    .. code-block:: c
+
+        // Parameter server object
+        rclc_parameter_server_t param_server;
+
+        // Initialize parameter server with default configuration
+        rcl_ret_t rc = rclc_parameter_server_init_default(&param_server, &node);
+
+        if (RCL_RET_OK != rc)
+        {
+            ... // Handle error
+            return -1;
+        }
+
+Memory requirements
+^^^^^^^^^^^^^^^^^^^
+
+The parameter server uses five services and an optional publisher. These need to be taken into account on the rmw-microxrcedds package memory configuration:
+
+.. code-block:: python
+
+    # colcon.meta example with memory requirements to use a parameter server
+    {
+        "names": {
+            "rmw_microxrcedds": {
+                "cmake-args": [
+                    "-DRMW_UXRCE_MAX_NODES=1",
+                    "-DRMW_UXRCE_MAX_PUBLISHERS=1",
+                    "-DRMW_UXRCE_MAX_SUBSCRIPTIONS=0",
+                    "-DRMW_UXRCE_MAX_SERVICES=5",
+                    "-DRMW_UXRCE_MAX_CLIENTS=0"
+                ]
+            }
+        }
+    }
+
+At runtime, the variable ``RCLC_PARAMETER_EXECUTOR_HANDLES_NUMBER`` defines the necessary number of handles required by a parameter server for the rclc Executor:
+
+.. code-block:: c
+
+    // Executor init example with the minimum RCLC executor handles required
+    rclc_executor_t executor = rclc_executor_get_zero_initialized_executor();
+    rc = rclc_executor_init(
+        &executor, &support.context,
+        RCLC_PARAMETER_EXECUTOR_HANDLES_NUMBER, &allocator);
+
+Humble: the variable is ``RCLC_PARAMETER_EXECUTOR_HANDLES_NUMBER`` has been renamed to ``RCLC_EXECUTOR_PARAMETER_SERVER_HANDLES``.
```

Comment:
We are supporting only humble, rephrase this paragraph and the previous so we use `RCLC_EXECUTOR_PARAMETER_SERVER_HANDLES`